src/main/java/core/basesyntax/Area.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Area {
+    public double getArea();
+}

```

Comment:
The interface name `Area` is a noun, which is typically good for classes and interfaces, but according to the checklist, interface names should not be verbs or simple nouns that don't reflect their behavior. Consider renaming this interface to something like `AreaCalculatable` to better convey its purpose.